### DISCUSSION TITLE
I have a problem.

### DISCUSSION THREAD

**Discussion by karimcubicazmy** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#756] [Q&A]
I am trying to access Google Assistant but I keep getting the same page.
Any way to fix this? 
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/110678019/183090278-620fe735-9fcb-4852-b47b-d1d2ed290ffd.png">

**Comment by im-coder-lg** [comment]
Check at #671, they got the solution in the comment history.
